kubernetes/cluster_deploy.go

```diff
@@ -54,23 +60,40 @@ type AppConfig struct {
 }
 
 type LoadtestConfig struct {
-	ReplicaCount                      int  `yaml:"replicaCount"`
-	NumTeams                          int  `yaml:"numTeams"`
-	NumChannelsPerTeam                int  `yaml:"numChannelsPerTeam"`
-	NumUsers                          int  `yaml:"numUsers"`
-	SkipBulkLoad                      bool `yaml:"skipBulkLoad"`
-	TestLengthMinutes                 int  `yaml:"testLengthMinutes"`
-	NumActiveEntities                 int  `yaml:"numActiveEntities"`
-	ActionRateMilliseconds            int  `yaml:"actionRateMilliseconds"`
-	ActionRateMaxVarianceMilliseconds int  `yaml:"actionRateMaxVarianceMilliseconds"`
+	ReplicaCount                      int           `yaml:"replicaCount"`
+	Image                             *ImageSetting `yaml:"image"`
+	NumTeams                          int           `yaml:"numTeams"`
+	NumChannelsPerTeam                int           `yaml:"numChannelsPerTeam"`
+	NumUsers                          int           `yaml:"numUsers"`
+	SkipBulkLoad                      bool          `yaml:"skipBulkLoad"`
+	TestLengthMinutes                 int           `yaml:"testLengthMinutes"`
+	NumActiveEntities                 int           `yaml:"numActiveEntities"`
+	ActionRateMilliseconds            int           `yaml:"actionRateMilliseconds"`
+	ActionRateMaxVarianceMilliseconds int           `yaml:"actionRateMaxVarianceMilliseconds"`
 }
 
 type ImageSetting struct {
 	Tag string `yaml:"tag"`
 }
 
 // TODO: Replace with an argument or config option when load test profiles are added
-const NUM_USERS = 10000
+const NUM_USERS = 30000
+
+const masterMySQLConfig = `
+[mysqld]
+log-bin
```

Comment:
Should we converge on `log_bin` (and `super_read_only`) for configuration files, vs. using the allowed (but confusing!) command line flavours here?

Reply:
Yeah good point, I just copied what they were using in the myqlha chart but even if they're not being consistent we should be